docs/user/FAQ.rst

```diff
@@ -357,3 +357,50 @@ layer, and that the number of features can be determined by
 ``X.shape[1]``. If those assumptions are not true for your case,
 adjust the code accordingly. A fully working example can be found
 on `stackoverflow <https://stackoverflow.com/a/60170023/1643939>`_.
+
+How do I implement a score method on the net that returns the loss?
+-------------------------------------------------------------------
+
+Sometimes, it is useful to be able to compute the loss of a net from within
+``skorch`` (*e.g.,* when a net is part of an ``sklearn`` pipeline). The function
```

Comment:
```suggestion
``skorch`` (e.g. when a net is part of an ``sklearn`` pipeline). The function
```
Or why should "e.g." be in italics?

Reply:
I'd italicized it because it's Latin, not English. But it does not matter on the whole, and I admit it is better to opt for consistency. I suppose better readability of the plaintext is a bonus of no italics.

Reply:
Okay, I didn't know that. As you said, for consistency (and since it's easier to type), let's do without.